bigcommerce/connection.py

```diff
@@ -53,6 +53,10 @@ def _run_method(self, method, url, data=None, query=None, headers=None):
         if headers is None:
             headers = {}
 
+        # Support v3
```

Comment:
@VDuda with this bit, the orders API no longer works, as this seems to have an explicit `/v2/` path as per the docs here:
https://developer.bigcommerce.com/api-reference/orders/orders-api/orders/getallorders

This bit of the code injects `catalog` in when it is not needed, the url for orders is `/store/{store_hash}/v2/orders`

Using the code this way would require making a fresh connection for a `v2` connection so we don't break the rest of the API

@bookernath am I missing anything with your API endpoints here?